sympy/simplify/tests/test_hyperexpand.py

```diff
@@ -48,8 +48,13 @@ def can_do(ap, bq, numerical=True, div=1, lowerplane=False):
     if not numerical:
         return True
     repl = {}
-    for n, a in enumerate(r.free_symbols - {z}):
-        repl[a] = randcplx(n)/div
+    randsyms = r.free_symbols - {z}
+    while randsyms:
+        # Only randomly generated parameters are checked.
```

Comment:
I first checked all parameters, also those that were entered in numeric form as arguments. But that made test_polynomial fail since it makes use of negative integers parameters that are explicitly set on purpose, not random.